nuxt.config.js

```
@@ -76,18 +84,34 @@ export default {
     '@nuxtjs/axios',
     '@nuxtjs/apollo',
     '@nuxtjs/auth-next',
+    [
+      'nuxt-gmaps',
+      {
+        key: 'AIzaSyCPS6SZlor8qxfpul-dKyN6566XG2R5dFM',
```

Read from env and fallback to hardcoded.

---

Resolved